### ISSUE TITLE
XP-Pen G640S having issues with the driver

### ISSUE THREAD

**Issue by Hokurikudo** [hawku/TabletDriver#1221]
I've downloaded the drivers as shown in the installation guide, but even though I've installed it correctly, it shows me this error on the bottom left, "Driver stopped. Restarting! Check console !!!" and when the console is checked it shows,

"Tablet init failed!
Possible fixes:

    1.Uninstall other tablet drivers.
    2.Stop other tablet driver-services."

I have uninstalled my previous driver which was the XP-Pen driver and I'm unsure why the hawku drivers won't work. Thank you so much for taking your time to read this! :)

Windows Version: Windows 7 (x32)

Tablet Driver version: V0.2.3

Tablet model: XP-Pen G640S

**Comment by jamesbt365** [comment]
read the readme

**Comment by Hokurikudo** [comment]
> read the readme

no driver for windows 32-bit?

**Comment by jamesbt365** [comment]
Wrong: https://github.com/OpenTabletDriver/OpenTabletDriver/actions/runs/2393682240
Also this driver doesn't have 32 bit binaries anyway. OpenTabletDriver does.

Though you really shouldn't be using 32 bit now, its very very outdated and limits what your computer can do by alot, unless you are on 32 bit processor i recommend you update to 64 bit.
